src/VisualStudio/Xaml/Impl/Features/OrganizeImports/XamlRemoveUnnecessaryImportsService.cs

```diff
@@ -29,5 +29,10 @@ public Task<Document> RemoveUnnecessaryImportsAsync(Document document, Cancellat
         {
             return _removeService.RemoveUnnecessaryNamespacesAsync(document, cancellationToken) ?? Task.FromResult(document);
         }
+
+        public Task<Document> RemoveUnnecessaryImportsFromCurrentContextAsync(Document document, Func<SyntaxNode, bool> predicate, CancellationToken cancellationToken)
```

Comment:
Why is this changing here? You didn't change IRemoveUnnecessaryImportsService, right?